docker-compose.yml

```
@@ -0,0 +1,29 @@
+version: "3"
+services:
+  node-dev:
+    build: 
+      context: .
+      dockerfile: Node.Dockerfile
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock 
+      - ~/.ssh:/home/.ssh
+      - .:/usr/app
+    working_dir: /usr/app
+    command: bash -c "npm install && npm run dev"
+    environment: 
+      - FIRESTORE_HOST="${FIRESTORE_HOST}"
+      - FIRESTORE_PROJECT_ID="${FIRESTORE_PROJECT_ID}"
```

Carregar isso do `.env` diretamente
```suggestion
   env_file: .env
```